### ISSUE TITLE
GitHub Codespaces shows "Error establishing a database connection"

### ISSUE THREAD

**Issue by joehoyle** [humanmade/altis-local-server#507]
Steps to reproduce:
1. Run codespaces against the altis-skeleton repo
2. Run `composer server start`
3. Visit the xxxx.githubpreview.dev URL
4. Observe "Error establishing a database connection"

![Screen Shot 2022-07-28 at 11 05 15 AM](https://user-images.githubusercontent.com/161683/181467124-0eb20715-3b6b-4c69-8be8-31eeb3bb34d2.png)

![Screen Shot 2022-07-28 at 11 05 12 AM](https://user-images.githubusercontent.com/161683/181467138-0eb85ef0-78a4-4aea-917e-bf58106f89c1.png)

![Screen Shot 2022-07-28 at 11 06 13 AM](https://user-images.githubusercontent.com/161683/181467328-9f45c630-9ea7-45e2-87a2-87d5056bc0d7.png)

**Comment by mikelittle** [comment]
@joehoyle 
I Noticed this in my testing too. I also saw that sometimes the ES container would also fail to start or take a really long time.
However, once I switched to using a 4-core Codespace, all these issue disappeared. I have started and stopped my codespace multiple times without seeing these issues.

I think the problem above was an artifact of trying to start 14 containers in a resouce-restricted environment.

I recommend this should be closed.
